api/index.js

```diff
@@ -40,5 +83,25 @@ api.get('/events', (req, res) => {
   res.send(events);
 });
 
+api.get('/stocks/:symbol', (req, res) => {
+  var Quandl = require('quandl');
+  var quandl = new Quandl({
+    auth_token: process.env.quandl,
```

Comment:
I thought it had been. So far I've been hard-coding the API key for testing and replacing it with the dotenv before making a commit.